assets/sass/main.scss

```diff
@@ -331,7 +331,7 @@
         background: #fff;
         border: solid 1px _palette(border);
         border-radius: _size(radius);
-        color: inherit;
```

Comment:
This should probably be squashed into the other color commit?

Reply:
You're right. I rebased the commits.